src/pages/resultsView/enrichments/GenericAssayEnrichmentsBoxPlot.tsx

```diff
@@ -0,0 +1,298 @@
+import * as React from 'react';
```

Comment:
How different is this from `ExpressionEnrichmentsBoxPlot.tsx`? can we combine them?